docs/source/common/lightning_module.rst

```diff
@@ -191,8 +195,9 @@ Under the hood, Lightning does the following (pseudocode):
         optimizer.step()
 
 
-Training Epoch-Level Metrics
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Train Epoch-Level Metrics
+=========================
```

Comment:
```suggestion
Epoch-Level Metrics during Training
==========================
```

This title is a bit confusing

Reply:
Might be better to have Epoch-Level Metrics and sub section for training, validation and test.

Reply:
Right now ste structure is like:
```
- Training
    - ...
    - epoch level metrics
    - ...
- Validation
    - ...
    - epoch level metrics
    - ...
- Testing
    - ... 
```
which I think is better because we have other subsections as well that belongs to each of the main heading.

Reply:
@rohitgr7 Could you update the section titles?